testonly/tree_fuzz_test.go

```diff
@@ -0,0 +1,110 @@
+//go:build go1.18
+
+package testonly
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/transparency-dev/merkle/proof"
+)
+
+// Compute and verify consistency proofs
+func FuzzConsistencyProof(f *testing.F) {
+	for size := 0; size <= 8; size++ {
+		for size2 := 0; size2 <= size; size2++ {
+			for size1 := 0; size1 <= size2; size1++ {
```

Comment:
I realise that you're following the naming in the files under test, but trying to understand this initialization loop order (size, size2, size1) derailed me. Perhaps `size`, `from`, `to` (or `start` and `end`) would be clearer in the tests?

Reply:
Good idea, done.